.travis.yml

```diff
@@ -26,6 +25,4 @@ before_script:
 script:
   - bundle exec rake rubocop
   - bundle exec rake test:validate
-  - bundle exec rake check:matching_docs
-  - npm run swagger2openapi
-  - npm run validate
+  - bundle exec rake documentation:generate
```

Comment:
The docs are currently being generated as a dependency of the tests. Is it necessary to generate it again?

Reply:
I can make doc generation to skip if a file already exist (with an option to force regeneration).